guides/release/getting-started/quick-start.md

```diff
@@ -218,22 +214,21 @@ The only difference is that now we've componentized our list into a version that
 You can see this in action if you create a new route that shows a different list of people.
 As an exercise for the reader,
 you may try to create a `programmers` route that shows a list of famous programmers.
-By re-using the `people-list` component, you can do it in almost no code at all.
+By re-using the `PeopleList` component, you can do it in almost no code at all.
 
 ## Click Events
 
 So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First add an `action` helper to the `li` in your `PeopleList` component.
 
-```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
```

Comment:
Not a blocker, but a question: What does `data-diff="-5,+6"` do? Googled this but couldn't find anything.

Reply:
It's how we define diffing, `-5` will show a removed line styling, and `+6` will show an added line styling.
In general I think it's super confusing and our color scheme isn't great, so I edited the code samples to not use diffs.

Reply:
Thank you for the explanation, much appreciated! Will have to take a closer look at diffing as get more familiar with this repo